src/atoms/Spacer/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
-import styles from './spacer.module.scss';
+import styles from './spacer.scss';
```

Comment:
Also, I suggest importing it as `spacing` for consistency with the other components